coral/src/app/layout/main-navigation/MainNavigation.tsx

```diff
@@ -134,8 +134,13 @@ function MainNavigation() {
               active={pathname.startsWith(Routes.USER_PROFILE)}
             />
             <MainNavigationLink
-              to={"/changePwd"}
+              to={
+                userInformationFeatureFlagEnabled
+                  ? Routes.USER_CHANGE_PASSWORD
+                  : "/changePwd"
+              }
               linkText={"Change password"}
+              active={pathname.startsWith(Routes.USER_CHANGE_PASSWORD)}
```

Comment:
not blocking for this PR because it's not TOO important, but usually we've a test case for that.

Reply:
I'll add it ^^

Reply:
But where? :o I don't see a test in MainNavigation that does it